lints/lint_old_root_ca_rsa_mod_less_than_2048_bits.go

```diff
@@ -26,7 +26,11 @@ func (l *rootCaModSize) CheckApplies(c *x509.Certificate) bool {
 }
 
 func (l *rootCaModSize) RunTest(c *x509.Certificate) (ResultStruct, error) {
-	mod := c.PublicKey.(*rsa.PublicKey).N
+	key, found := c.PublicKey.(*rsa.PublicKey)
```

Comment:
This is super confusing in general though. Because it looks like `lint_old_root_ca_rsa_mod_less_than_2048_bits.go` checks this in the `CheckApplies` function. Whereas `lint_rsa_mod_less_than_2048_bits.go` defines this in `EffectiveDate`. Is there a reason this isn't done in the same manner for both lints?